grammar.js

```diff
@@ -1191,18 +1208,24 @@ module.exports = grammar({
       field('body', $.block)
     ),
 
-    _reserved_identifier: $ => alias(choice(
-      'open',
-      'module',
-      'record'
-    ), $.identifier),
+    _reserved_identifier: $ => prec(-3, alias(
+      choice(
+        'open',
+        'module',
+        'record',
+        'with',
+        'yield',
+        'sealed',
```

Comment:
Do we need `non-sealed` here?

Reply:
I'm not sure if it's valid, if it is lmk